std_msgs/Header header

uint32 id
uint32 time_stamp
float32 range
float32 azimuth
float32 elevation
float32 range_rate
float32 amplitude
uint32 is_aggregate
int32 tracking_status
uint32 age
uint32 last_seen
bool is_stationary
bool is_ghost